x-pack/plugins/alerting/common/alert.ts

```diff
@@ -97,3 +97,10 @@ export interface AlertsHealth {
     timestamp: string;
   };
 }
+
+export interface ActionVariable {
```

Comment:
Since I was updating the `ActionVariable`, I resolved [this issue](https://github.com/elastic/kibana/issues/90191) to remove the duplicate definition.